examples/cli-create2/index.js

```diff
@@ -1,10 +1,10 @@
-const { ConfigVariablesInitializer, files, scripts, stdout } = require('zos');
+const { ConfigManager, files, scripts, stdout } = require('zos');
```

Comment:
Could you add a note to the `lib/changelog`, indicating this breaking change, so we don't forget about it when we add this feature to the CLI changelog?